translations/fr.po

```diff
@@ -627,3 +531,84 @@ msgstr "Générant {} pour {}"
 #: ui.py:126
 msgid "{} generated for {}"
 msgstr "{} généré pour {}"
+
```

Comment:
This chunk of commented change is unnecessary as mentioned before.

Reply:
I don't understand this one, doesn't seem :126  is commented.

Reply:
Got it.  It is lines 534-614 that are commented (and must be preserved). Just don't understand Poedit behaviour. Looking at config.py I remain clueless.

Reply:
These language names are used at here:
https://github.com/xxyzz/WordDumb/blob/316e19fba35ff678065b5284d9a2872e398d3c87/config.py#L422-L429

`lemma_languages` and `gloss_languages` are used for the drop down button text at here:
https://github.com/xxyzz/WordDumb/blob/316e19fba35ff678065b5284d9a2872e398d3c87/config.py#L438-L445

Reply:
Ah, I see. Thanks for explaining.